themes/custom/assets/css/custom.css

```
@@ -4,6 +4,21 @@
   background: #4584b6 !important;
 }
 
+/* Tabla de contenidos local */
+
+@media screen and (min-width: 108em) and (max-width: 200em) {
+    .contents.topic {
```

estamos identando con 2 espacios :D
```suggestion
  .contents.topic {
```